modules/schedulers/src/main/java/org/mule/modules/schedulers/cron/CronScheduler.java

```diff
@@ -170,7 +174,10 @@ public void stop() throws MuleException
     {
         try
         {
-            quartzScheduler.pauseAll();
+            if (quartzScheduler != null)
```

Comment:
Is this change needed? the scheduler is nullified at dispose, which is executed after the stop.